### ISSUE TITLE
Uh...?

### ISSUE THREAD

**Issue by OctoTheBitch** [thesupersonic16/HedgeModManager#320]
HMM Info:
```
    Version: 7.6-0
    Args: 
    StartDir: C:\Users\daigo\Desktop\Sonic Generations\
    Process Level: Administrator
    Game: Sonic Generations
    SteamGame: Sonic Generations
```
[HMM_Snapshot_00092021185106.txt](https://github.com/thesupersonic16/HedgeModManager/files/7219801/HMM_Snapshot_00092021185106.txt)

Exception:
```
    Type: NullReferenceException
    Message: Object reference not set to an instance of an object.
    Source: HedgeModManager
    Function: Void UI_MouseDoubleClick(System.Object, System.Windows.Input.MouseButtonEventArgs)
    StackTrace: 
       at HedgeModManager.MainWindow.UI_MouseDoubleClick(Object sender, MouseButtonEventArgs e) in C:\Users\SuperSonic16\source\repos\HedgeModManager\HedgeModManager\UI\MainWindow.xaml.cs:line 891
       at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
       at System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
       at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
       at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
       at System.Windows.Controls.Control.HandleDoubleClick(Object sender, MouseButtonEventArgs e)
       at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
       at System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
       at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
       at System.Windows.UIElement.ReRaiseEventAs(DependencyObject sender, RoutedEventArgs args, RoutedEvent newEvent)
       at System.Windows.UIElement.OnMouseDownThunk(Object sender, MouseButtonEventArgs e)
       at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
       at System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
       at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
       at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
       at System.Windows.UIElement.RaiseTrustedEvent(RoutedEventArgs args)
       at System.Windows.Input.InputManager.ProcessStagingArea()
       at System.Windows.Input.InputManager.ProcessInput(InputEventArgs input)
       at System.Windows.Input.InputProviderSite.ReportInput(InputReport inputReport)
       at System.Windows.Interop.HwndMouseInputProvider.ReportInput(IntPtr hwnd, InputMode mode, Int32 timestamp, RawMouseActions actions, Int32 x, Int32 y, Int32 wheel)
       at System.Windows.Interop.HwndMouseInputProvider.FilterMessage(IntPtr hwnd, WindowMessage msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
       at System.Windows.Interop.HwndSource.InputFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
       at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
       at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
       at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
       at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
       at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
       at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
       at MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)
       at System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
       at System.Windows.Application.RunDispatcher(Object ignore)
       at System.Windows.Application.RunInternal(Window window)
       at HedgeModManager.HedgeApp.Main(String[] args) in C:\Users\SuperSonic16\source\repos\HedgeModManager\HedgeModManager\HedgeApp.xaml.cs:line 330
    InnerException: 
```

**Comment by thesupersonic16** [comment]
I was not able to reproduce this issue, but I have pushed a possible fix 6dbcd7edfdee20f79b96fa8de68dff90c875f53a, this will be included in the next HedgeModManager release.

If you can provide any details on how to reproduce this bug, I could test to make sure its fixed.

**Comment by OctoTheBitch** [comment]
I'm not sure at all, whenever I load up a mod with code injection it al always seems to crash on startup.

**Comment by thesupersonic16** [comment]
Fixed in [7.6-1](https://github.com/thesupersonic16/HedgeModManager/releases/tag/7.6-1)
